README.md

```diff
@@ -1 +1,10 @@
-# HikingJournal
\ No newline at end of file
+# HikingJournal
+
+## Description
+
+HikingJournal is simple journal mobile(only android for now) application providing easy to use way of saving your hiking routes, memorable places location etc. Saved only on your phone.
+
+## Features
+- Taking photo creates journal entry with your current location coordinates.
+
+## Milestones
```

Comment:
I am not sure about having empty section (only title).
Maybe we can add some text here about roadmap.
Or maybe we can delete the section title and add it again when we have ideas about milestones?